posthog/hogql/database/schema/cohort_people.py

```diff
@@ -23,11 +23,7 @@ def select_from_cohort_people_table(requested_fields: Dict[str, List[str]]):
 
     table_name = "raw_cohort_people"
 
-    requested_fields = {"person_id": ["person_id"], "cohort_id": ["cohort_id"], **requested_fields}
-
-    fields: List[ast.Expr] = [
-        ast.Alias(alias=name, expr=ast.Field(chain=[table_name] + chain)) for name, chain in requested_fields.items()
-    ]
+    fields: List[ast.Expr] = [ast.Field(chain=[table_name] + chain) for name, chain in requested_fields.items()]
```

Comment:
This line was there so that we always request the person and cohort fields. Without them, some queries and/or joins will just fail. 

Thus, if you now select just the person_id, or the cohort_id or just something else that's not those two, will this fail?

Reply:
Seems to work the same with those two fields always bolted back on... e.g. `select team_id from cohort_people` works either way.

Is there an obvious test case to try?

Reply:
(I've put the line ensuring they're present back in)

Reply:
Hmm... actually I'll take that back. The issue I was remembering is valid only in the case of automatic joins (e.g. `events.person.bla`). The join constraint would fail if we didn't always select the fields we're going to use in the constraint. However... that's not relevant here, so insert homer bush gif here 🤣